The release automation cannot find the npm binary

### Description

The following error is triggered when running the release automation:
```
Run npm ci
/runner/_work/_temp/31706a96-2d57-439e-b81b-840061ce0bfc.sh: line 1: npm: command not found
Error: Process completed with exit code 127.
```

### Steps to reproduce

1. run the release automation workflow.

### Additional context

_No response_

### Hedera network

mainnet

### Version

v0.49.0

### Operating system

Linux

--- a/.github/workflows/release-automation.yml
+++ b/.github/workflows/release-automation.yml
@@ -152,6 +152,11 @@ jobs:
           gpg_private_key: ${{ secrets.GPG_KEY_CONTENTS }}
           passphrase: ${{ secrets.GPG_KEY_PASSPHRASE }}
 
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+
       - name: Install dependencies
         run: npm ci
 

--- a/.github/workflows/release-automation.yml
+++ b/.github/workflows/release-automation.yml
@@ -153,7 +153,7 @@ jobs:
           passphrase: ${{ secrets.GPG_KEY_PASSPHRASE }}
 
       - name: Set up Node.js
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
         with:
           node-version: '18'
 

